languages/csharp/exercises/concept/nullability/.docs/after.md

```diff
@@ -0,0 +1,56 @@
+Sometimes we need to make it so that variables have no particular
+value, i.e. they are empty.  In C#, this corresponds to the literal
+`null`. 
+
+In this exercise, we saw the definition of nullable types, and how the
+compiler and runtime of C# help us dealing with `null` values.
+
+At compilation time, the operator `?` will declare a variable as being
+*nullable*. The compiler will then try to help us avoiding calling
+methods on possibly `null` variables, by raising warnings. You can use
+the operator `!` to avoid warnings in places we are sure a nullable
+variable is not null, but the compiler cannot detect it. Finally, we
+can use the operator `??` to provide a default value for a nullable
+variable.
+
+At run time, calling any method or property on a
+`null` value throws a `NullReferenceException` exception.
+That is why it is important to always check if a nullable variable
+is `null` before calling methods on its value.
+
+# Important changes in C# 8.0
+
+Sometimes, we need to make sure that some variables are never
+`null`. This simplifies code because it won't need to handle
+`NullReferenceException`s or provide extra provisions for `null`
+values.
+
+Before C# 8.0, reference types were nullable by default. For example,
+a variable of type `string` may contain a `null` value, even it is not
+declared as `string?`.
+
+For more information, refer to [this
+documentation][nullable-csharp-8].
+
+# Additional Resources
```

Comment:
The other C# exercises link to the resources from the text instead of having a separate list. See [this example](https://github.com/exercism/v3/blob/master/languages/csharp/exercises/concept/strings/.docs/after.md). Could you also apply that here?

Reply:
Ok! I will work on that

Reply:
Great!